### PR TITLE
[angular] Update Angular version 10.1.0

### PR DESCRIPTION
**Pull request by qmonmert**
New version of **Angular 10.1.0**

Commands :
``` 
ng update
ng update @angular/cli @angular/core rxjs
``` 

---

Please make sure the below checklist is followed for Pull Requests.

-   [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
-   [x] Tests are added where necessary
-   [x] Documentation is added/updated where necessary
-   [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (bellow reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

File: generators/client/templates/angular/tsconfig.json.ejs [hunk 1]
```diff
@@ -17,14 +17,35 @@
...
+    "strict": true,
+    "noImplicitReturns": true,
+    "noFallthroughCasesInSwitch": true,
+    "suppressImplicitAnyIndexErrors": true,
+    "skipLibCheck": true,
+    "outDir": "target/classes/static/app",
+    "lib": ["es7", "dom"],
+    "baseUrl": "./",
+    "paths": {
+      "app/*": ["src/main/webapp/app/*"]
```
Comment (mshima) [inline generators/client/templates/angular/tsconfig.json.ejs:36]:
```suggestion
      "app/*": ["<%= CLIENT_MAIN_SRC_DIR %>app/*"]
```

File: generators/client/templates/angular/tsconfig.json.ejs [hunk 2]
```diff
@@ -17,14 +17,35 @@
...
+    "declaration": false,
+    "forceConsistentCasingInFileNames": true
+  },
+  "angularCompilerOptions": {
+    "strictInjectionParameters": true,
+    "fullTemplateTypeCheck": true,
+    "strictTemplates": true,
+    "preserveWhitespaces": true
+  },
+  "exclude": ["src/test/javascript/cypress"]
```
Comment (mshima) [inline generators/client/templates/angular/tsconfig.json.ejs:49]:
```suggestion
  "exclude": ["<%= CLIENT_TEST_SRC_DIR %>cypress"]
```

File: generators/client/templates/angular/tsconfig.json.ejs [hunk 3]
```diff
@@ -17,14 +17,35 @@
...
+    "moduleResolution": "node",
+    "sourceMap": true,
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "strict": true,
+    "noImplicitReturns": true,
+    "noFallthroughCasesInSwitch": true,
+    "suppressImplicitAnyIndexErrors": true,
+    "skipLibCheck": true,
+    "outDir": "target/classes/static/app",
```
Comment (mshima) [inline generators/client/templates/angular/tsconfig.json.ejs:32]:
```suggestion
    "outDir": "<%= DIST_DIR %>app",
```

**Review by mshima — APPROVED** [review]

File: generators/client/templates/angular/tsconfig.json.ejs (hunk 1 above)
Comment (qmonmert) [reply generators/client/templates/angular/tsconfig.json.ejs:36]: yes thanks @mshima

**Comment by DanielFran** [comment]
Thanks @qmonmert

**Pull request merged**